docker-storage-setup.sh

```diff
@@ -240,12 +236,8 @@ write_storage_config_file () {
     if ! storage_options=$(get_devicemapper_config_options); then
       return 1
     fi
-  elif [ "$STORAGE_DRIVER" == "overlay" ];then
-    if ! storage_options=$(get_overlay_config_options); then
-      return 1
-    fi
-  elif [ "$STORAGE_DRIVER" == "overlay2" ];then
-    if ! storage_options=$(get_overlay2_config_options); then
+  else
+    if ! storage_options=$(get_config_options $STORAGE_DRIVER); then
```

Comment:
This seems to be partial cleanup. We have another function get_devicemapper_config_options() and get_config_options() seems to imply that it works for all graph drivers.

So either create get_config_options() which works for all graph drivers, or rename this function to something which represents that this is overlay specific.